### PR TITLE
Add FH 216

### PR DESCRIPTION
**Pull request by simonjulianl**

### CODE REVIEW

File: content/post/2022-01-28-friday-hacks-215.md [hunk 1]
```diff
@@ -24,4 +26,10 @@ In this talk, Jet will share his personal journey in RL projects, and share some
...
 
-## 2) TBC
+## 2) Linear Complexity Transformers
+
+Transformer-based language models, such as BERT and GPT, have made significant advances in the field of natural language processing, outperforming previously existing deep learning models. However, they suffer with the issue of quadratic complexity. This talk combines the techniques of linear complexity attention and sequence parallelism to (hopefully) enable handling longer input sequences than was previously possible using the original Transformer architecture.
+
+#### Speaker Profile
+
+_Chaitanya_ is an NUS Hackers coreteam member, and CS undergrad who enjoys learning new stuff. He is yet to discover what part of technology truly interests him, but his technical interests lie in algorithms and distributed systems. He wishes to work on impactful tech stuff which will positively affect people's lives.
```
Comment (kouyk) [inline content/post/2022-01-28-friday-hacks-215.md:35]: He _has_ yet

File: layouts/shortcodes/friday_hack_header.html [hunk 2]
```diff
@@ -1,7 +1,9 @@
 <blockquote>
     <strong>Date/Time</strong>: Friday, {{ .Get "date" }}{{ if not (in (.Get "date") " at ") }} at 7:00pm{{ end }}<br />
     <strong>Venue</strong>: {{ .Get "venue" | safeHTML }} <br />
-    <strong>Free {{ .Get "food" | default "pizza" }} is served before the talks at 6:30pm!</strong> <br />
+    {{ with .Get "food" }}
+      <strong>Free {{ . }} is served before the talks at 6:30pm!</strong> <br />
+    {{ end }}
```
Comment (kouyk) [inline layouts/shortcodes/friday_hack_header.html:6]: Wild idea, i wonder how much more we can automate the youtube link and remove the food info after the event is over.

**Review by kouyk — APPROVED** [review]
LGTM, just a minor typo that's all

File: content/post/2022-01-28-friday-hacks-215.md (hunk 1 above)
Comment (simonjulianl) [reply content/post/2022-01-28-friday-hacks-215.md:35]: yup

File: layouts/shortcodes/friday_hack_header.html (hunk 2 above)
Comment (simonjulianl) [reply layouts/shortcodes/friday_hack_header.html:6]: The food info is fine but youtube link meaning it needs to scrape or have youtube API, which um slightly mafan

**Pull request merged**